jupyter_server_proxy/handlers.py

```diff
@@ -373,6 +373,82 @@ async def proxy(self, host, port, proxied_path):
             )
         else:
             client = httpclient.AsyncHTTPClient()
+        # check if the request is stream request
+        proxy_streaming = self.request.headers.get('Accept')
```

Comment:
Let's rename this variable to `accept_header` to more descriptively describe what it is

Reply:
fixed